classes/module/Module.php

```diff
@@ -3185,7 +3185,7 @@ public function isAdminLegacyContext()
      */
     public function isSymfonyContext()
     {
-        return !defined('ADMIN_LEGACY_CONTEXT') && $this->context->controller instanceof AdminLegacyLayoutControllerCore;
+        return !defined('ADMIN_LEGACY_CONTEXT') && defined('_PS_ADMIN_DIR_');
```

Comment:
should be even better with `return !$this->isAdminLegacyContext() && defined('PS_ADMIN_DIR');` to remove duplicate :)

Reply:
edited